lib/rubocop/cop/rails/pluck_in_where.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop identifies places where `pluck` is used in `where` query methods
+      # and can be replaced with `select`.
+
```

Comment:
Class comments need to be connected :-)

```suggestion
      # and can be replaced with `select`.
      #
```

Reply:
Ah, sure, that was unintentional :)